nautobot/extras/api/serializers.py

```diff
@@ -112,6 +113,7 @@
     NestedTagSerializer,
     NestedWebhookSerializer,
     NestedJobHookSerializer,
+    NestedRoleSerializer,
```

Comment:
Can you move these two to their appropriate alphabetized position in the list please?